lib/ssl/src/ssl_crl_cache.erl

```diff
@@ -56,7 +56,7 @@ select(Issuer, {{_Cache, Mapping},_}) ->
     case ssl_pkix_db:lookup(Issuer, Mapping) of
 	undefined ->
 	    [];
-	CRLs ->
+	[CRLs] ->
```

Comment:
This looks wrong, mapping is a bag and thus can return a list of more than one element?

Reply:
I have never looked at this code :-)  I just thought that change looked suspicious, the other looks ok to me.

What you say is true but depending on the table type you look into an ets:lookup() will return a list of one element 
or a list of several in case of duplicates.

The other uses calls lookup on the cache an 'set' table which will return the list of CRLs inside a list, 
but the change above does a lookup on 'Mappings' which is 'bag' and therefore I guess the intention
is to return the list.

But I don't know the code enough to say which is right here, therefore the question mark, but wanted to
note it before I forget about it.

Reply:
It was too long ago that I worked with this code to remember for sure what is the intent in this particular case. Although I do remember that for the common case of this DB abstraction it is the intent to remove the list around the only value returned
of the lookup.  Probably a test will help us refresh the understanding of the implementation.